frappe/website/js/WebForm.js

```diff
@@ -0,0 +1,78 @@
+export default class WebForm {
```

Comment:
lets stay with small case for filenames? We can't change all the files, so consistency is better with lowercase!